src/components/list/tests/DogList.test.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import DogList from '../DogList';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('DogList component', () => {
+  afterEach(() => cleanup());
+  it('renders DogList', () => {
+    const { asFragment } = render(
+      <MemoryRouter>
+        <DogList dogs={[]} />
```

Comment:
you could import in your dogs list.json and spread it here for a "more real" test. Either is fine.